src/test/java/com/marklogic/kafka/connect/sink/FailedConversionErrorReporterTest.java

```diff
@@ -0,0 +1,62 @@
+package com.marklogic.kafka.connect.sink;
+
+import com.marklogic.client.document.DocumentWriteOperation;
+import org.apache.kafka.connect.header.Headers;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FailedConversionErrorReporterTest extends AbstractIntegrationSinkTest {
+
+    Throwable reportedException = null;
+    SinkRecord reportedSinkRecord = null;
+    final private String someValidJsonToSignalAContrivedException = "{ \"A\": \"a\"}";
+
+    @Test
+    void sinkRecordWithNullValueCausesCallToErrorReporter() {
```

Comment:
Just need one more assertion here - that the good record is written. To make that easy, can do the following:

1. Add e.g. `MarkLogicSinkConfig.DOCUMENT_COLLECTIONS, "good-data"` to the task config
2. Add `assertCollectionSize("good-data", 1)` to the end of the test, or rather after the flush occurs

Reply:
drat, forgot this one.